src/main/java/com/sap/cloudfoundry/client/facade/adapters/RawApplicationLog.java

```diff
@@ -16,28 +15,34 @@
 public abstract class RawApplicationLog implements Derivable<ApplicationLog> {
 
     @Value.Parameter
-    public abstract Envelope getEnvelope();
+    public abstract ApplicationLogEntity getLog();
 
     @Override
     public ApplicationLog derive() {
-        Envelope envelope = getEnvelope();
-        LogMessage logMessage = envelope.getLogMessage();
+        ApplicationLogEntity log = getLog();
         return ImmutableApplicationLog.builder()
-                                      .applicationGuid(logMessage.getApplicationId())
-                                      .message(logMessage.getMessage())
-                                      .timestamp(fromLogTimestamp(logMessage))
-                                      .messageType(fromLogMessageType(logMessage))
-                                      .sourceId(logMessage.getSourceInstance())
-                                      .sourceName(logMessage.getSourceType())
+                                      .applicationGuid(log.getSourceId())
```

Comment:
Is the logs source id the same as app id?

Reply:
Yes. It's the "source" of the emitted log event.